src/Ticket.php

```diff
@@ -2701,6 +2701,17 @@ public function rawSearchOptions()
 
         $tab = array_merge($tab, $this->getSearchOptionsMain());
 
+        $tab[] = [
+            'id'                 => '70',
+            'table'              => $this->getTable(),
+            'field'              => '_virtual_age',
+            'datatype'           => 'specific',
+            'name'               => __('Time since opening'),
+            'massiveaction'      => false,
+            'nosearch'           => true,
+            'additionalfields'   => ['id']
```

Comment:
You can add `entities_id` and `date` into this array to fetch all the data you need at once.
You will then no longer need to reload the ticket from the database in the `getSpecificValueToDisplay` function.